homeassistant/components/hassmpris/translations/en.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
Please remove the translation file.
Recent changes have made it redundant.

Reply:
Will do so.  Right away.

Translation removed.